### ISSUE TITLE
[Bug]: it doesnt start

### ISSUE THREAD

**Issue by gskale17** [EncryptSL/LiteEco#54]
### What happened?

A bug happened!

### Plugin Version

1.3.3-SNAPSHOT

### You detected problem on this server platform ?

PaperMC

### Relevant log output

```shell
[18:03:15] [ServerMain/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[18:03:21] [ServerMain/INFO]: Loaded 7 recipes
[18:03:24] [Server thread/INFO]: Starting minecraft server version 1.19.4
[18:03:24] [Server thread/INFO]: Loading properties
[18:03:24] [Server thread/INFO]: This server is running CraftBukkit version 3755-Spigot-7d7b241-b346a5f (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT)
[18:03:25] [Server thread/INFO]: Debug logging is disabled
[18:03:25] [Server thread/INFO]: Server Ping Player Sample Count: 12
[18:03:25] [Server thread/INFO]: Using 4 threads for Netty based IO
[18:03:25] [Server thread/INFO]: Default game type: SURVIVAL
[18:03:25] [Server thread/INFO]: Generating keypair
[18:03:25] [Server thread/INFO]: Starting Minecraft server on *:26878
[18:03:25] [Server thread/INFO]: Using epoll channel type
[18:03:28] [Server thread/INFO]: [LiteEco] Loading 5 libraries... please wait
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/com/zaxxer/HikariCP/5.0.1/HikariCP-5.0.1.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/slf4j/slf4j-api/2.0.0-alpha1/slf4j-api-2.0.0-alpha1.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.8.0/kotlin-stdlib-jdk8-1.8.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlin/kotlin-stdlib/1.8.0/kotlin-stdlib-1.8.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlin/kotlin-stdlib-common/1.8.0/kotlin-stdlib-common-1.8.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/annotations/13.0/annotations-13.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlin/kotlin-stdlib-jdk7/1.8.0/kotlin-stdlib-jdk7-1.8.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/exposed/exposed-core/0.40.1/exposed-core-0.40.1.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlin/kotlin-reflect/1.7.20/kotlin-reflect-1.7.20.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/kotlinx/kotlinx-coroutines-core-jvm/1.6.4/kotlinx-coroutines-core-jvm-1.6.4.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/org/jetbrains/exposed/exposed-jdbc/0.40.1/exposed-jdbc-0.40.1.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/com/squareup/okhttp3/okhttp/4.10.0/okhttp-4.10.0.jar
[18:03:28] [Server thread/INFO]: [LiteEco] Loaded library /server/libraries/com/squareup/okio/okio-jvm/3.0.0/okio-jvm-3.0.0.jar
[18:03:28] [Server thread/INFO]: [LuckPerms] Loading LuckPerms v5.4.40
[18:03:29] [Server thread/INFO]: [TreeAssist] Loading TreeAssist v7.3.39
[18:03:29] [Server thread/INFO]: [TreeAssist] Loading main config file: /config.yml
[18:03:29] [Server thread/INFO]: [OpenInv] Loading OpenInv v4.3.0
[18:03:29] [Server thread/INFO]: [Vault] Loading Vault v1.7.3-b131
[18:03:29] [Server thread/INFO]: [PlaceholderAPI] Loading PlaceholderAPI v2.11.4-DEV-null
[18:03:29] [Server thread/INFO]: [ProtocolLib] Loading ProtocolLib v5.0.0-SNAPSHOT-642
[18:03:33] [Server thread/INFO]: [SetHomes] Loading SetHomes v1.3.0
[18:03:33] [Server thread/INFO]: [Citizens] Loading Citizens v2.0.31-SNAPSHOT (build 3097)
[18:03:33] [Server thread/INFO]: [SkinsRestorer] Loading SkinsRestorer v14.2.10
[18:03:33] [Server thread/INFO]: [ChestShop] Loading ChestShop v3.12.2 (build 353)
[18:03:33] [Server thread/INFO]: [SuperVanish] Loading SuperVanish v6.2.12
[18:03:33] [Server thread/INFO]: [LiteEco] Loading LiteEco v1.3.3-SNAPSHOT
[18:03:33] [Server thread/INFO]: [LiteEco] Configuration database.db exist !
[18:03:33] [Server thread/INFO]: [LiteEco] Configuration config.yml is latest !
[18:03:33] [Server thread/INFO]: HikariPool-1 - Starting...
[18:03:33] [Server thread/INFO]: HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@83d7299
[18:03:33] [Server thread/INFO]: HikariPool-1 - Start completed.
[18:03:34] [Server thread/INFO]: [LuckPerms] Enabling LuckPerms v5.4.40
[18:03:35] [Server thread/INFO]:         __    
[18:03:35] [Server thread/INFO]:   |    |__)   LuckPerms v5.4.40
[18:03:35] [Server thread/INFO]:   |___ |      Running on Bukkit - CraftBukkit
[18:03:35] [Server thread/INFO]: 
[18:03:35] [Server thread/INFO]: [LuckPerms] Loading configuration...
[18:03:35] [Server thread/INFO]: [LuckPerms] Loading storage provider... [H2]
[18:03:37] [Server thread/INFO]: [LuckPerms] Loading internal permission managers...
[18:03:37] [Server thread/INFO]: [LuckPerms] Performing initial data load...
[18:03:37] [Server thread/INFO]: [LuckPerms] Successfully enabled. (took 3840ms)
[18:03:37] [Server thread/INFO]: [Vault] Enabling Vault v1.7.3-b131
[18:03:38] [Server thread/INFO]: [Vault] [Permission] SuperPermissions loaded as backup permission system.
[18:03:38] [Server thread/INFO]: [Vault] Enabled Version 1.7.3-b131
[18:03:38] [Server thread/INFO]: [LuckPerms] Registered Vault permission & chat hook.
[18:03:38] [Server thread/INFO]: [ProtocolLib] Enabling ProtocolLib v5.0.0-SNAPSHOT-642
[18:03:38] [Server thread/INFO]: [LiteEco] Enabling LiteEco v1.3.3-SNAPSHOT
[18:03:38] [Server thread/INFO]: [LiteEco] PlaceholderAPI hook initialized
[18:03:38] [Server thread/INFO]: [PlaceholderAPI] Successfully registered expansion: liteeco [1.0.2]
[18:03:38] [Server thread/INFO]: [LiteEco] Registered Vault like a service.
[18:03:38] [Server thread/INFO]: [LiteEco] Treasury not found, for better experience please download Treasury or Vault.
[18:03:39] [Server thread/INFO]: [LiteEco] Please download update of plugin LiteEco your version: 1.3.3-SNAPSHOT > updated version: 1.3.4-SNAPSHOT
[18:03:39] [Server thread/INFO]: [LiteEco] Registering commands with Cloud Command Framework !
[18:03:40] [Server thread/ERROR]: Error occurred while enabling LiteEco v1.3.3-SNAPSHOT (Is it up to date?)
encryptsl.cekuj.net.cloud.bukkit.BukkitCommandManager$BrigadierFailureException: Could not initialize Brigadier mappings. Reason: version too high ()
	at encryptsl.cekuj.net.cloud.bukkit.BukkitCommandManager.registerBrigadier(BukkitCommandManager.java:343) ~[?:?]
	at encryptsl.cekuj.net.cloud.paper.PaperCommandManager.registerBrigadier(PaperCommandManager.java:125) ~[?:?]
	at encryptsl.cekuj.net.LiteEco.registerCommands(LiteEco.kt:86) ~[?:?]
	at encryptsl.cekuj.net.LiteEco.onEnable(LiteEco.kt:65) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[spigot-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:536) ~[spigot-1.19.4-R0.1-SNAPSHOT.jar:3755-Spigot-7d7b241-b346a5f]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:450) ~[spigot-1.19.4-R0.1-SNAPSHOT.jar:3755-Spigot-7d7b241-b346a5f]
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:220) ~[spigot-1.19.4-R0.1-SNAPSHOT.jar:3755-Spigot-7d7b241-b346a5f]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:975) ~[spigot-1.19.4-R0.1-SNAPSHOT.jar:3755-Spigot-7d7b241-b346a5f]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.19.4-R0.1-SNAPSHOT.jar:3755-Spigot-7d7b241-b346a5f]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
[18:03:40] [Server thread/WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[18:03:40] [Server thread/WARN]: The server will make no attempt to authenticate usernames. Beware.
[18:03:40] [Server thread/WARN]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[18:03:40] [Server thread/WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[18:03:40] [Server thread/INFO]: Preparing level "world"
[18:03:40] [Server thread/INFO]: -------- World Settings For [world] --------
[18:03:40] [Server thread/INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true / Isa false
[18:03:40] [Server thread/INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Di 128 / Other 64
[18:03:40] [Server thread/INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1 Hopper Can Load Chunks: false
[18:03:40] [Server thread/INFO]: Mob Spawn Range: 6
[18:03:40] [Server thread/INFO]: Experience Merge Radius: 3.0
[18:03:40] [Server thread/INFO]: Cactus Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Cane Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Melon Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Mushroom Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Pumpkin Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Sapling Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Beetroot Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Carrot Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Potato Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Wheat Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: NetherWart Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Vine Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Cocoa Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Bamboo Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: SweetBerry Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Kelp Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: TwistingVines Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: WeepingVines Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: CaveVines Growth Modifier: 100%
[18:03:40] [Server thread/INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911 Nether: 30084232 Mansion: 10387319 Fossil: 14357921 Portal: 34222645
[18:03:40] [Server thread/INFO]: Max TNT Explosions: 100
[18:03:40] [Server thread/INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[18:03:40] [Server thread/INFO]: Item Despawn Rate: 6000
[18:03:40] [Server thread/INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[18:03:40] [Server thread/INFO]: Item Merge Radius: 2.5
[18:03:40] [Server thread/INFO]: View Distance: 5
[18:03:40] [Server thread/INFO]: Simulation Distance: 5
[18:03:40] [Server thread/INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[18:03:40] [Server thread/INFO]: Zombie Aggressive Towards Villager: true
[18:03:40] [Server thread/INFO]: Nerfing mobs spawned from spawners: false
[18:03:41] [Server thread/INFO]: -------- World Settings For [world_nether] --------
[18:03:41] [Server thread/INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true / Isa false
[18:03:41] [Server thread/INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Di 128 / Other 64
[18:03:41] [Server thread/INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1 Hopper Can Load Chunks: false
[18:03:41] [Server thread/INFO]: Mob Spawn Range: 6
[18:03:41] [Server thread/INFO]: Experience Merge Radius: 3.0
[18:03:41] [Server thread/INFO]: Cactus Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Cane Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Melon Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Mushroom Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Pumpkin Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Sapling Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Beetroot Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Carrot Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Potato Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Wheat Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: NetherWart Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Vine Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Cocoa Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Bamboo Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: SweetBerry Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Kelp Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: TwistingVines Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: WeepingVines Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: CaveVines Growth Modifier: 100%
[18:03:41] [Server thread/INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911 Nether: 30084232 Mansion: 10387319 Fossil: 14357921 Portal: 34222645
[18:03:41] [Server thread/INFO]: Max TNT Explosions: 100
[18:03:41] [Server thread/INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[18:03:41] [Server thread/INFO]: Item Despawn Rate: 6000
[18:03:41] [Server thread/INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[18:03:41] [Server thread/INFO]: Item Merge Radius: 2.5
[18:03:41] [Server thread/INFO]: View Distance: 5
[18:03:41] [Server thread/INFO]: Simulation Distance: 5
[18:03:41] [Server thread/INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[18:03:41] [Server thread/INFO]: Zombie Aggressive Towards Villager: true
[18:03:41] [Server thread/INFO]: Nerfing mobs spawned from spawners: false
[18:04:00] [Server thread/INFO]: -------- World Settings For [world_the_end] --------
[18:04:00] [Server thread/INFO]: Entity Activation Range: An 32 / Mo 32 / Ra 48 / Mi 16 / Tiv true / Isa false
[18:04:00] [Server thread/INFO]: Entity Tracking Range: Pl 48 / An 48 / Mo 48 / Mi 32 / Di 128 / Other 64
[18:04:00] [Server thread/INFO]: Hopper Transfer: 8 Hopper Check: 1 Hopper Amount: 1 Hopper Can Load Chunks: false
[18:04:00] [Server thread/INFO]: Mob Spawn Range: 6
[18:04:00] [Server thread/INFO]: Experience Merge Radius: 3.0
[18:04:00] [Server thread/INFO]: Cactus Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Cane Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Melon Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Mushroom Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Pumpkin Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Sapling Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Beetroot Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Carrot Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Potato Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Wheat Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: NetherWart Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Vine Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Cocoa Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Bamboo Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: SweetBerry Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Kelp Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: TwistingVines Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: WeepingVines Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: CaveVines Growth Modifier: 100%
[18:04:00] [Server thread/INFO]: Custom Map Seeds:  Village: 10387312 Desert: 14357617 Igloo: 14357618 Jungle: 14357619 Swamp: 14357620 Monument: 10387313 Ocean: 14357621 Shipwreck: 165745295 End City: 10387313 Slime: 987234911 Nether: 30084232 Mansion: 10387319 Fossil: 14357921 Portal: 34222645
[18:04:00] [Server thread/INFO]: Max TNT Explosions: 100
[18:04:00] [Server thread/INFO]: Tile Max Tick Time: 50ms Entity max Tick Time: 50ms
[18:04:00] [Server thread/INFO]: Item Despawn Rate: 6000
[18:04:00] [Server thread/INFO]: Allow Zombie Pigmen to spawn from portal blocks: true
[18:04:00] [Server thread/INFO]: Item Merge Radius: 2.5
[18:04:00] [Server thread/INFO]: View Distance: 5
[18:04:00] [Server thread/INFO]: Simulation Distance: 5
[18:04:00] [Server thread/INFO]: Arrow Despawn Rate: 1200 Trident Respawn Rate:1200
[18:04:00] [Server thread/INFO]: Zombie Aggressive Towards Villager: true
[18:04:00] [Server thread/INFO]: Nerfing mobs spawned from spawners: false
[18:04:00] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:13] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:14] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:14] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:15] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:15] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:16] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:16] [Worker-Main-1/INFO]: Preparing spawn area: 54%
[18:04:17] [Server thread/INFO]: Time elapsed: 16748 ms
[18:04:17] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:25] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:26] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:26] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:27] [Worker-Main-1/INFO]: Preparing spawn area: 4%
[18:04:28] [Server thread/INFO]: Time elapsed: 10573 ms
[18:04:28] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:34] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:35] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:35] [Worker-Main-1/INFO]: Preparing spawn area: 0%
[18:04:35] [Server thread/INFO]: Time elapsed: 7932 ms
[18:04:36] [Server thread/INFO]: [TreeAssist] Enabling TreeAssist v7.3.39
[18:04:36] [Server thread/INFO]: [TreeAssist] sending full Metrics! You can deactivate this in the config.yml
[18:04:36] [Server thread/INFO]: [TreeAssist] debugging: off
[18:04:36] [Server thread/INFO]: [OpenInv] Enabling OpenInv v4.3.0
[18:04:36] [Thread-15/INFO]: [TreeAssist] Checking for updates...
[18:04:36] [Server thread/INFO]: [PlaceholderAPI] Enabling PlaceholderAPI v2.11.4-DEV-null
[18:04:36] [Server thread/INFO]: [PlaceholderAPI] Fetching available expansion information...
[18:04:36] [Server thread/INFO]: [SetHomes] Enabling SetHomes v1.3.0
[18:04:36] [Server thread/INFO]: [SetHomes] Hooked into LuckPerms!
[18:04:36] [Server thread/INFO]: [Citizens] Enabling Citizens v2.0.31-SNAPSHOT (build 3097)
[18:04:36] [Server thread/INFO]: [Citizens] Loading external libraries
[18:04:37] [Server thread/INFO]: [PlaceholderAPI] Successfully registered expansion: citizensplaceholder [1.0.0]
[18:04:37] [Server thread/INFO]: [Citizens] Loaded economy handling via Vault.
[18:04:37] [Server thread/INFO]: [SkinsRestorer] Enabling SkinsRestorer v14.2.10
[18:04:38] [Server thread/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1mDetected Minecraft [33;1mv1_19_R3[32;1m, using [33;1mMappingSpigotSkinRefresher[32;1m.[0;39m
[18:04:38] [Server thread/INFO]: [ChestShop] Enabling ChestShop v3.12.2 (build 353)
[18:04:39] [Server thread/INFO]: [ChestShop] Found locales ar, it, es, el, hu, fr, pl, cs, pt_br, ja, sv, da, es_mx, sr, tr, nl, zh_tw, af, ko, no, vi, ro, he, ru, ca, zh, de, fi, en, uk
[18:04:39] [Server thread/INFO]: [ChestShop] Using LiteEco as the Economy provider now.
[18:04:39] [Server thread/INFO]: [ChestShop] Vault loaded!
[18:04:39] [Server thread/INFO]: [ChestShop] Auto-updater is disabled. If you want the plugin to automatically download new releases then set 'TURN_OFF_UPDATES' to 'false' in your config.yml!
[18:04:39] [Server thread/INFO]: [SuperVanish] Enabling SuperVanish v6.2.12
[18:04:39] [Server thread/INFO]: [PlaceholderAPI] Successfully registered expansion: supervanish [6.2.12]
[18:04:39] [Server thread/INFO]: [SuperVanish] Hooked into PlaceholderAPI
[18:04:39] [Server thread/INFO]: [SuperVanish] Hooked into Citizens
[18:04:39] [Server thread/WARN]: Could not register alias sell because it contains commands that do not exist: economyshopgui:sellall $1-
[18:04:40] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[18:04:40] [Server thread/INFO]: [PlaceholderAPI] Placeholder expansion registration initializing...
[18:04:40] [Server thread/INFO]: 0 placeholder hook(s) registered!
[18:04:40] [Server thread/INFO]: Done (60.217s)! For help, type "help"
[18:04:40] [Server thread/INFO]: Starting GS4 status listener
[18:04:40] [Server thread/INFO]: Thread Query Listener started
[18:04:40] [Query Listener #1/INFO]: Query running on 0.0.0.0:9898
[18:04:40] [Server thread/INFO]: JMX monitoring enabled
[18:04:41] [User Authenticator #1/INFO]: UUID of player citabir is b46faff7-edb6-3941-9216-eaefbf9dd416
[18:04:41] [Server thread/INFO]: [Citizens] Loaded 5 NPCs.
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m----------------------------------------------[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    +==================+[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    |   SkinsRestorer  |[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    |------------------|[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    |  [34;1mStandalone Mode[0;39m[32;1m |[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    +==================+[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m----------------------------------------------[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[36;1m    Current version: [32;1m14.2.10[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m    This is the latest version![0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [33;1m[[32;22mSkinsRestorer[33;1m] [0;39m[32;1m----------------------------------------------[0;39m
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [Vault] Checking for Updates ... 
[18:04:41] [Server thread/INFO]: [SuperVanish] Your current version of SuperVanish is outdated - New version: '6.2.16'; Currently: '6.2.12'
[18:04:41] [Craft Scheduler Thread - 1/INFO]: [Vault] No new version available
[18:04:41] [Server thread/INFO]: com.mojang.authlib.GameProfile@7d27b234[id=b46faff7-edb6-3941-9216-eaefbf9dd416,name=citabir,properties={},legacy=false] (/5.27.19.184:10377) lost connection: Disconnected
```

**Comment by EncryptSL** [comment]
Send version output from server. I need info about server.
/ver -> command.

**Comment by EncryptSL** [comment]
Maybe fixed in 1.3.5 i can't replicate your problem. 

But i updated libs and another stuff for working you can try.

**Comment by Nekroz0701** [comment]
This server is running CraftBukkit version 3762-Spigot-7d7b241-2270366 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT) 

That the ver command
